gulp-tasks/utils/build-browser-typescript.js

```diff
@@ -24,50 +24,45 @@ const processScript = (scriptPath, relativePath, tsConfigPath, destDir) => {
              name: path.basename(scriptPath),
            },
            plugins: [
-             // The plugin can only read tsconfig from the cwd so instead,
-             // we read in the file and set the compiler options directly
-             // in the plugin options (hence the `...tsconfig.compilerOptions`)
-             // Then `tsconfig: false` disables any tsconfig.json file reading
-             typescriptPlugin({
-               ...tsconfig.compilerOptions,
-               tsconfig: false,
-               typescript: require('typescript'),
-             }),
-             uglifyPlugin({}, esMinify),
+             sourcemapPlugin(),
+             // uglifyPlugin({}, esMinify),
```

Comment:
Should this be back?

Reply:
For now I'd say no - just to see if the sourcemap is even remotely useful.